### PR TITLE
Fix grant types hint wordings

### PR DESCRIPTION
**Pull request by JayaShakthi97**
### Purpose
- Fix grant types hint wordings
- Fix README file miswording

### Related Issues
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/extensions/configs/application.tsx [hunk 1]
```diff
@@ -70,6 +70,7 @@ export const applicationConfig: ApplicationConfig = {
     excludeSubjectClaim: false,
     inboundOIDCForm: {
         disabledGrantTypes: [],
+        sdkName: "Our SDKs",
```
Comment (JayaShakthi97) [inline apps/console/src/extensions/configs/application.tsx:73]: Replace with product name config

Comment (JayaShakthi97) [inline apps/console/src/extensions/configs/application.tsx:73]: Replace with product name config

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 2]
```diff
@@ -1460,11 +1460,15 @@ export const console: ConsoleNS = {
                                         label: "(openid scope not allowed)"
                                     },
                                     implicit: {
-                                        hint: "Using the implicit grant with public applications is not recommended.",
+                                        hint: "Using the implicit grant with public applications is not recommended. " +
+                                            "{{sdkName}} adhere to security best practices, " +
```
Comment (JayaShakthi97) [inline modules/i18n/src/translations/en-US/portals/console.ts:1464]: Product name

File: apps/console/src/extensions/configs/application.tsx (hunk 1 above)
Comment (JayaShakthi97) [reply apps/console/src/extensions/configs/application.tsx]: Fixed with d8b5df7a47ad41492c41e05bce2df78e4db93abd

Comment (JayaShakthi97) [reply apps/console/src/extensions/configs/application.tsx]: Fixed with d8b5df7a47ad41492c41e05bce2df78e4db93abd

Comment (JayaShakthi97) [reply apps/console/src/extensions/configs/application.tsx]: Fixed with d8b5df7a47ad41492c41e05bce2df78e4db93abd

File: modules/i18n/src/translations/en-US/portals/console.ts (hunk 2 above)
Comment (JayaShakthi97) [reply modules/i18n/src/translations/en-US/portals/console.ts]: Fixed with d8b5df7a47ad41492c41e05bce2df78e4db93abd

**Review by thivi — APPROVED** [review]

File: apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx [hunk 3]
```diff
@@ -622,17 +622,16 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
-                    "password.hint") + " " + "Asgardeo SDKs adhere to security best practices, and do not " +
-                    "implement the password grant.";
+                    "password.hint", { productName: config.ui.productName });
             case ApplicationManagementConstants.CLIENT_CREDENTIALS_GRANT:
                 return t("console:develop.features.applications.forms.inboundOIDC.fields.grant.children." +
                         "client_credential.hint");
             case ApplicationManagementConstants.REFRESH_TOKEN_GRANT:
-                return "Refresh token grant type should only be selected along with the Code grant type.";
+                return t("console:develop.features.applications.forms.inboundOIDC.fields.grant.validation." + 
+                    "refreshToken");
```
[the remark this answers is not in the document]
Comment (JayaShakthi97) [reply apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx:634]: i18n strings were already there for all the languages. What I did was fixing a key name typo in French file. (`vvalidation --> validation`)

**Pull request merged**